packages/sdk/contractkit/src/wrappers/FederatedAttestations.ts

```diff
@@ -1,8 +1,164 @@
 import { FederatedAttestations } from '../generated/FederatedAttestations'
-import { BaseWrapper } from './BaseWrapper'
+import { BaseWrapper, proxyCall, proxySend } from './BaseWrapper'
 
-// TODO ASv2 -- add params or delete config if there are none
-// & delete other commented-out usages of this Config in CK
-// export interface FederatedAttestationsConfig {}
+export class FederatedAttestationsWrapper extends BaseWrapper<FederatedAttestations> {
+  /**
+   * @notice Returns identifiers mapped to `account` by signers of `trustedIssuers`
+   * @param account Address of the account
+   * @param trustedIssuers Array of n issuers whose identifier mappings will be used
+   * @return countsPerIssuer Array of number of identifiers returned per issuer
+   * @return identifiers Array (length == sum([0])) of identifiers
+   * @dev Adds identifier info to the arrays in order of provided trustedIssuers
+   * @dev Expectation that only one attestation exists per (identifier, issuer, account)
+   */
+  lookupIdentifiers: (
+    account: string,
```

Comment:
if these are suppose to be addresses I suggest created a `type Address = string` and then setting all that must be an address (not just any random string) to that type for readability